avocado/utils/lv_utils.py

```diff
@@ -54,6 +56,38 @@ def get_diskspace(disk):
     raise LVException('Error in finding disk space')
 
 
+def get_device_total_space(device):
+    """Get the total device size.
+
+    :param str device: name of the device/disk to find the total size
+    :returns: size in bytes
+    :rtype: str
+    :raises: :py:class:`LVException` on failure to find disk space
+    """
+    try:
+        with open('/sys/class/block/{}/size'.format(device), 'r') as fp:
+            blocks = int(fp.read())
+            return blocks * 512
+    except FileNotFoundError:
+        raise LVException('Device not found.')
+
+
+def get_devices_total_space(devices):
+    """Get the total size of given device(s)/disk(s).
+
+    :param list devices: list with the names of devices separated with space.
+    :returns: sizes in bytes
+    :rtype: generator
+    :raises: :py:class:`LVException` on failure to find disk space
+    """
+    size = 0
+    for device in devices.split(" "):
+        size = size + int(get_device_total_space(device))
```

Comment:
Unless I'm missing something, you don't need to convert to int, right?

`get_device_total_space()` is already returning a `int`.

Reply:
`get_diskspace()` was returning `str`, and I think @Naresh-ibm has gone for keeping the same function intact for `get_device_total_space()`